prover/crates/lib/circuit_prover_service/src/witness_vector_generator/witness_vector_generator_metadata_loader.rs

```diff
@@ -81,3 +81,34 @@ impl WitnessVectorMetadataLoader for HeavyWitnessVectorMetadataLoader {
             .await
     }
 }
+
+/// Simple MetadataLoader.
+///
+/// This loader will pick any job available, regardless of its type.
+#[derive(Debug)]
+pub struct SimpleWitnessVectorMetadataLoader {
+    pod_name: String,
+    protocol_version: ProtocolSemanticVersion,
+}
+
+impl SimpleWitnessVectorMetadataLoader {
```

Comment:
I wonder if `Simple` is a good prefix... Why not have It without it? Although I'm fine if it's done after heavy/light removal